src/main/java/org/openlmis/buq/domain/buq/Rejection.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * This program is part of the OpenLMIS logistics management information system platform software.
+ * Copyright © 2017 VillageReach
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details. You should have received a copy of
+ * the GNU Affero General Public License along with this program. If not, see
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org.
+ */
+
+package org.openlmis.buq.domain.buq;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+import javax.persistence.CascadeType;
+import javax.persistence.CollectionTable;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.openlmis.buq.domain.BaseTimestampedEntity;
+
+
+@Entity
+@Getter
+@Setter
+@Table(name = "rejections")
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class Rejection extends BaseTimestampedEntity {
+
+  @ElementCollection
+  @CollectionTable(name = "rejection_rejection_reasons",
+          joinColumns = @JoinColumn(name = "rejectionId"))
+  private List<UUID> rejectionReasons;
+
+  private String generalComments;
+
+  @OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+  BottomUpQuantificationStatusChange statusChange;
```

Comment:
if this field intentionally has a default access modifier instead of a private, the PR looks OK.

Reply:
it wasn't meant to be public, thanks for pointing that out